packages/favorites/src/FavoriteHeart.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { HeartOutlined, HeartFilled, Spinner } from './Icons';
+import css from './FavoriteHeart.module.scss';
+import { useFavorites } from './FavoritesContext';
+
+export const FavoriteHeart = ({
+  id,
+  name,
+  onChange,
+  onMouseDown,
+}: {
+  id: string;
+  name: string;
+  onChange?: (isFavorited: boolean, event: React.ChangeEvent<HTMLInputElement>) => void;
+  onMouseDown?: (event: React.MouseEvent<HTMLInputElement, MouseEvent>) => void;
+}): JSX.Element => {
+  const { isFavorited, isDisabled, isActiveMutation, toggleFavorite } = useFavorites(id);
+
+  const handleChange = (e: React.ChangeEvent<HTMLInputElement>) => {
```

Comment:
I'm personally not a fan of single letter variables. I'm good with using it if no one else has issues

Reply:
Fixed